### PR TITLE
Some hints

### PR DESCRIPTION
**Pull request by NoZiL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+### Requirements
+
+Python 3 is required
+
+### Dependencies
+
+Install dependencies
+
+```
+Make init
```
Comment (GaelTouquet) [inline README.md:14]: Stylé!

File: init.sh [hunk 2]
```diff
@@ -1,3 +1,6 @@
 export GEOMETRY=$PWD
 export PYTHONPATH=$GEOMETRY/..:$PYTHONPATH
-export PATH=$GEOMETRY:$PATH
\ No newline at end of file
+export PATH=$GEOMETRY:$PATH
```
Comment (GaelTouquet) [inline init.sh:3]: It seems that this is not working on windows, read somewhere there is an easy solution using puthon's os library. #4

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+matplotlib==2.2.5
```
Comment (GaelTouquet) [inline requirements.txt:1]: Cool!

File: README.md (hunk 1 above)
Comment (NoZiL) [reply README.md:14]: la puissance du Makefile en C tu en fais tout le temps :)

File: requirements.txt (hunk 3 above)
Comment (NoZiL) [reply requirements.txt:1]: yep and you should keep using it, when you install a dep via pip, you have to add it here (sorry pip doesn't seem to do that automatically but pipenv does maybe that's worth looking into)

**Pull request merged**